src/go/k8s/README.md

```diff
@@ -34,12 +35,35 @@ export KUBECONFIG=your/path/to/kubeconfig.yaml
 kind create cluster --config kind.yaml
 ```
 
-You can simply deploy the Redpanda operator by running the following command
+In order to have validating webhook the cert manager needs to be
+installed. You can do this by running the following command:
+
+```
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
+```
+
+The cert manager needs around 1 minute to be ready. The Redpanda
+operator will create Issuer and Certificate custom resource. The
+webhook of cert-manager will prevent from creating mentioned
+resources. To verify that cert manager is ready for handling
+new request please run the following command:
+
+```
+kubectl wait pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=10m --for condition=ready
```

Comment:
this is incorrect, it's not enough to wait for the pod, that's why we're using the cert-manager-verifier in tests, because it's not that easy. https://cert-manager.io/docs/installation/kubernetes/#verifying-the-installation

let's just link here instead.

Reply:
Ok, changed